src/ripple/basics/ScopedLock.h

```diff
@@ -23,116 +23,50 @@
 
 #ifndef RIPPLE_BASICS_SCOPEDLOCK_H_INCLUDED
 #define RIPPLE_BASICS_SCOPEDLOCK_H_INCLUDED
-
```

Comment:
Ultra minor nit.  I usually expect empty vertical space between the include guard and the namespace.  If I'm reading the diff correctly, that vertical space went away?